netbox_branching/database.py

```diff
@@ -21,6 +21,11 @@ def _get_db(self, model, **hints):
             warnings.warn(f"Routing database query for {model} before branching support is initialized.")
             return
 
+        # Bail if the model does not support branching
```

Comment:
Not for this PR, but might be nice in the future to make this a utility function: model_supports_branching - I could see it might be useful in the future to potentially check elsewhere and it isolates the logic in a nice place / makes it more readable.

Reply:
+1. and it would definitely be useful if a plugin could declare its models branching-aware (or not). That way I could avoid having to tell users it's either branching or NetBox DNS at the moment.